src/measure_theory/measure/measure_space.lean

```diff
@@ -2603,6 +2603,22 @@ begin
   exact ne_of_lt (measure_lt_top _ _)
 end
 
+lemma ae_eq_univ_iff_measure_eq [is_finite_measure μ] (hs : null_measurable_set s μ) :
+  s =ᵐ[μ] univ ↔ μ s = μ univ :=
```

Comment:
See `ae_eq_of_subset_of_measure_ge`.

Reply:
Thanks, I've golfed the proof using `ae_eq_of_subset_of_measure_ge`.